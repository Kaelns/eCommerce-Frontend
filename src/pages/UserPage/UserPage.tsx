import { TitleTypography } from '@/shared/ui/elements/typography/TitleTypography';

export function UserPage() {
  return <TitleTypography>User Profile Page</TitleTypography>;
}
