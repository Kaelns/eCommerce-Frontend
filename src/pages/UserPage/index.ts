export { UserPage } from '@/pages/UserPage/UserPage';
