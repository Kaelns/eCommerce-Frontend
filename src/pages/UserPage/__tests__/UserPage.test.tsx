import { UserPage } from '@/pages/UserPage/UserPage';
import { render } from '@testing-library/react';

describe('Given UserPage component: ', () => {
  it('when rendered, should match snapshot', () => {
    const { asFragment } = render(<UserPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
