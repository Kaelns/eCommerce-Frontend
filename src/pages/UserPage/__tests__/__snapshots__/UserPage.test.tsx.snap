// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Given UserPage component:  > when rendered, should match snapshot 1`] = `
<DocumentFragment>
  <h5
    class="MuiTypography-root MuiTypography-h5 css-1yej70y-MuiTypography-root"
  >
    User Profile Page
  </h5>
</DocumentFragment>
`;
